fastlane_bot/events/exchanges/uniswap_v3.py

```diff
@@ -30,11 +30,11 @@ def add_pool(self, pool: Pool):
         self.pools[pool.state["address"]] = pool
 
     def get_abi(self):
-        return UNISWAP_V3_POOL_ABI if self.exchange_name not in [PANCAKESWAP_V3_NAME, AGNI_V3_NAME] else PANCAKESWAP_V3_POOL_ABI
+        return UNISWAP_V3_POOL_ABI if self.exchange_name not in [PANCAKESWAP_V3_NAME, AGNI_V3_NAME, FUSIONX_V3_NAME] else PANCAKESWAP_V3_POOL_ABI
 
     @property
     def get_factory_abi(self):
-        return UNISWAP_V3_FACTORY_ABI if self.exchange_name not in [PANCAKESWAP_V3_NAME, AGNI_V3_NAME] else PANCAKESWAP_V3_FACTORY_ABI
```

Comment:
technically, any `get_x()` method should not use an '@property' decorator, we should be following the example set by the base class for any abstract methods. However, it appears that there is only one usage of this method in the entire codebase, and that it's expecting '@property' behavior, so for the purposes of this PR I will approve this, but a new issue should be opened to correct this.